src/natcap/invest/cli.py

```diff
@@ -82,17 +77,25 @@ def build_model_list_json():
         A string representation of the JSON object.
 
     """
+    from natcap.invest import LOCALE_DIR
+    translation = gettext.translation(
+        'messages',
+        languages=[locale_code],
+        localedir=LOCALE_DIR,
+        # fall back to a NullTranslation, which returns the English messages
+        fallback=True)
+
     json_object = {}
-    for model_name, model_data in model_metadata.MODEL_METADATA.items():
-        json_object[model_data.model_title] = {
-            'model_name': model_name,
-            'aliases': model_data.aliases
+    for model_id, model_spec in model_id_to_spec.items():
+        json_object[translation.gettext(model_spec['model_name'])] = {
```

Comment:
This is a little bit different way of translating with gettext than we've used before, but I think it's a lot cleaner. Instead of reloading each model module with a new language, we're just getting the translation of the specific string that needs to be translated. This is easy to do here since we only need the `model_name`.

Reply:
This made sense to try now because, with the old approach and without the names centralized in `model_metadata.py`, we would have to reload _all_ model modules in the new language, just to get the `model_name` string from each.